escodegen.js

```diff
@@ -983,7 +983,13 @@
             result.push('[');
         }
 
-        result.push(this.generateExpression(expr, Precedence.Sequence, E_TTT));
+        result.push(this.generateExpression(
+            expr,
+            expr.type === Syntax.SequenceExpression
+                ? Precedence.Primary
+                : Precedence.Sequence,
+            E_TTT
+        ));
```

Comment:
```suggestion
        result.push(this.generateExpression(expr, Precedence.Assignment, E_TTT));
```

See https://tc39.es/ecma262/#prod-ComputedPropertyName.

Reply:
Thank you, https://github.com/estools/escodegen/pull/415/commits/13e3eb826b50ebcb5dd10efd9515f22bf9f73c3a